Tasks/DeployAzureResourceGroup/VirtualMachine.ts

```diff
@@ -1,8 +1,9 @@
 /// <reference path="../../definitions/node.d.ts" /> 
 /// <reference path="../../definitions/vsts-task-lib.d.ts" /> 
-  
+
 import path = require("path");
 import tl = require("vsts-task-lib/task");
+import mgExtManager = require("./MachineGroupAgentExtensionManager");
```

Comment:
Please make sure that using current directory as base directory is norm in task code.